libs/platform/src/lib/components/form/multi-combobox/pipes/highlight.pipe.ts

```diff
@@ -0,0 +1,45 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { MatchingStrategy, MultiComboboxConfig } from '../multi-combobox.config';
+
+// TODO: refactor it with HighlightPipe from Combobox and SearchHighlightPipe
```

Comment:
Is there has issue create ? If not - please create one & put its number here to have a reference.